x/errors.go

```diff
@@ -1,51 +1,16 @@
 package x
 
 import (
-	"fmt"
-
 	"github.com/iov-one/weave/errors"
 )
 
-// ABCI Response Codes
-const (
-	CodeInvalidCurrency uint32 = 30
-	CodeInvalidCoin            = 31
-)
-
+// Reserved codes 130~139
 var (
-	errInvalidCurrency = fmt.Errorf("Invalid currency code")
-	errOutOfRange      = fmt.Errorf("Overflow coin range")
-	errMismatchedSign  = fmt.Errorf("Mismatched sign")
-	errInvalidWallet   = fmt.Errorf("Invalid wallet")
+	InvalidCurrencyErr = errors.Register(130, "invalid currency code")
```

Comment:
As commented on https://github.com/iov-one/weave/issues/309#issuecomment-465022957

can we use the ErrXxx format for all new errors, at least the ones we add on a package-specific basis

Reply:
In the errors package we have `Err` suffix but 👍 to change this back to prefix.

Reply:
@alpe this is an outdated diff, all done